.github/workflows/security-orchestration.yml

```
@@ -0,0 +1,279 @@
+          name: security-reports-${{ github.run_id }}
+          path: |
+            reports/security/
+            safety_report.json
+            bandit_report.json
+            semgrep_report.json
+            outdated_packages.json
+          retention-days: 30
+      
+      - name: Security Check Summary
+        if: always()
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:38: Double quote to prevent globbing and word splitting

(shellcheck)